### PR TITLE
kubecost: Bump kubecost to v1.75.1

### PR DESCRIPTION
**Pull request by gracedo**
**What type of PR is this?**

chore

**What this PR does/ why we need it**:

This fix was merged and released https://github.com/kubecost/cost-analyzer-helm-chart/pull/773. Bumping kubecost to the latest version which includes the fix, and no need to disable PSP now

**Which issue(s) this PR fixes**:

https://jira.d2iq.com/browse/D2IQ-74182

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
kubecost: Enable PodSecurityPolicy
```

**Checklist**

* [x] *If a chart is changed, the chart version is correctly incremented.*
* [x] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: stable/kubecost/values.yaml [hunk 1]
```diff
@@ -57,9 +57,6 @@ cost-analyzer:
     enabled: true
     # storageClass: "-"
 
-  podSecurityPolicy:
-    enabled: false
```
Comment (jr0d) [inline stable/kubecost/values.yaml:61]: Assuming the default is `true1`?

**Review by jr0d — APPROVED** [review]

**Review by branden — APPROVED** [review]

**Review by s12chung — APPROVED** [review]

File: stable/kubecost/values.yaml (hunk 1 above)
Comment (joejulian) [reply stable/kubecost/values.yaml:61]: yes. True is the default.

**Review by joejulian — APPROVED** [review]

**Review by alejandroEsc — APPROVED** [review]
lgtm

File: stable/kubecost/values.yaml (hunk 1 above)
Comment (gracedo) [reply stable/kubecost/values.yaml:61]: maybe not `true1`, but definitely `true` 😉

**Pull request merged**